package.json

```diff
@@ -23,8 +24,8 @@
     "lint:eslint": "eslint . --ext js,cjs,mjs --ignore-path .gitignore",
     "dev": "vite",
     "build:docs": "vite build && cp CNAME site",
-    "eik:login": "eik login",
-    "eik:publish": "eik publish",
+    "eik:login": "npx @eik/cli login",
+    "eik:publish": "npx @eik/cli publish",
     "eik:alias": "eik pkg-alias",
```

Comment:
I don't think this is used anywhere, so I guess we can remove it.

Reply:
Don’t we use them here https://github.com/warp-ds/elements/pull/4/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R32